src/status_im/chat/models/input.cljs

```diff
@@ -48,10 +48,11 @@
         cursor      (+ at-sign-idx (count primary-name) 2)]
     (rf/merge
      cofx
-     {:db                   (-> db
-                                (assoc-in [:chats/cursor chat-id] cursor)
-                                (assoc-in [:chats/mention-suggestions chat-id] nil))
-      :set-text-input-value [chat-id new-text text-input-ref]}
+     {:db (-> db
+              (assoc-in [:chats/cursor chat-id] cursor)
+              (assoc-in [:chats/mention-suggestions chat-id] nil))
+      ;:set-text-input-value [chat-id new-text text-input-ref]
```

Comment:
This event was causing a big problem:

When the contact suggested to mention was pressed below the input:

![image](https://user-images.githubusercontent.com/90291778/229266651-d0c4fe25-4f52-41ba-8139-9869fcfbe71e.png)

The content of the input was duplicated, e.g. if the input had: `Hola @mentioned-contact` it became `Hola @mentioned-contact Hola @mentioned-contact`.

This happened because this event takes the input reference to sets its value (through its `.setNativeProps` method)  and, simultaneously,  children were taking its value from `(rf/sub [:chat/input-with-mentions])`.

Temporarily I commented this call, but it's still a problem since this is only fixed on Android, atm I don't have access to an iOS simulator/device to test